src/main/java/com/example/sharemind/comment/repository/CommentCustomerRepositoryImpl.java

```diff
@@ -0,0 +1,54 @@
+package com.example.sharemind.comment.repository;
+
+import com.example.sharemind.comment.domain.Comment;
+import com.example.sharemind.comment.domain.QComment;
+import com.example.sharemind.counselor.domain.Counselor;
+import com.example.sharemind.post.content.PostStatus;
+import com.example.sharemind.post.domain.QPost;
+import com.querydsl.core.types.dsl.BooleanExpression;
+import com.querydsl.jpa.impl.JPAQueryFactory;
+import lombok.RequiredArgsConstructor;
+
+import java.util.List;
+
+@RequiredArgsConstructor
+public class CommentCustomerRepositoryImpl implements CommentCustomerRepository {
+
+    private final JPAQueryFactory jpaQueryFactory;
+    QComment comment = QComment.comment;
+    QPost post = QPost.post;
+
+    @Override
+    public List<Comment> findAllByCounselorAndIsActivatedIsTrue(Counselor counselor, Boolean filter, Long postId, int size) {
+
+        BooleanExpression predicate = commentByCounselor(counselor)
+                .and(filterCondition(filter))
+                .and(postIdCondition(postId));
+
+        return jpaQueryFactory
+                .selectFrom(comment)
+                .where(predicate)
```

Comment:
, 로 이어주면 따로 and를 명시하지 않아도 and 조건으로 묶입니다! (이미 알고 계셨다면 쓰루해주세요...ㅎㅎㅎ)

```suggestion
                .where(
                  commentByCounselor(counselor),
                  filterCondition(filter),
                  postIdCondition(postId)
                )
```

Reply:
헉 몰랐습니다~ 감사합니다!